hr_holidays_calendar_event_privacy/tests/test_hr_leave_calendar_event.py

```diff
@@ -24,15 +27,15 @@ def _new_leave_request(cls, date_from, date_to):
     def test_calendar_event_privacy(self):
         self.assertEqual(self.leave_type.calendar_event_privacy, "confidential")
         leave = self._new_leave_request(
-            fields.Date.today() + timedelta(days=12),
-            fields.Date.today() + timedelta(days=13),
+            fields.Date.today() + timedelta(days=2),
```

Comment:
I don't get why are you changing this data.

Reply:
@pedrobaeza I'm not sure, but this may fix errors like this one: https://github.com/OCA/hr-holidays/actions/runs/8315045201/job/22752936701?pr=114 (PR #114)

Reply:
OK, seems reasonable.